src/community/backlog/index.md.njk

```diff
@@ -366,12 +449,20 @@ Here is a list of the components, patterns and updates currently on the Design S
         To do
       </td>
     </tr>
+    <tr>
+      <td class="govuk-table__cell">
+        <a href="https://github.com/alphagov/govuk-design-system-backlog/issues/168">Messages</a>
+      </td>
+      <td class="govuk-table__cell govuk-body-s" style="text-align: right">
+        Proposed
+      </td>
+    </tr>
     <tr>
       <td class="govuk-table__cell">
         <a href="https://github.com/alphagov/govuk-design-system-backlog/issues/30">Modal dialogue</a>
       </td>
       <td class="govuk-table__cell govuk-body-s" style="text-align: right">
-        In progress
+        To do
```

Comment:
Is this correct?

Reply:
It's 'todo' in the projects board (these changes are from scraping the board) so if it's not the project board should be updated

Reply:
I've moved the card to 'In Progress' – is it easy to update this PR to match?

Reply:
Sure thing